gapis/api/gles/tweaker.go

```diff
@@ -91,6 +91,12 @@ func (t *tweaker) glDisable(ctx context.Context, name GLenum) {
 	}
 }
 
+func (t *tweaker) glDisableVertexAttribArray(ctx context.Context, l AttributeLocation) {
+	t.doAndUndo(ctx,
```

Comment:
Strictly speaking, this needs the "if not already in the desired state", otherwise calling this already disabled array, will incorrectly enable it after the tweaker is reverted.
I know that nn this case you do the check in the caller.

Reply:
Done